zipkin-collector/rabbitmq/pom.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2015-2017 The OpenZipkin Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+    in compliance with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the License
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+    or implied. See the License for the specific language governing permissions and limitations under
+    the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.zipkin.java</groupId>
+    <artifactId>zipkin-collector</artifactId>
+    <version>2.0.2-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>zipkin-collector-rabbitmq</artifactId>
+  <name>Collector: RabbitMQ</name>
+	<description>Zipkin span collector for RabbitMQ transport</description>
+
+	<properties>
+    <main.basedir>${project.basedir}/../..</main.basedir>
+
+    <rabbitmq-client.version>4.2.0</rabbitmq-client.version>
+  </properties>
+
+	<dependencies>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>zipkin</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.slf4j</groupId>
```

Comment:
is this an indirect dep of rabbit? if not, let's not use it (as it is unnecessary). use JUL instead.

Reply:
`slf4j-api` is the RabbitMQ client library's [only required dependency](https://github.com/rabbitmq/rabbitmq-java-client/blob/master/pom.xml#L625-L629) but we don't need to use it ourselves. I'm not sure why I did... probably just habit.